bin/dev/kube-ready-state-check.sh

```diff
@@ -108,7 +108,7 @@ fi
 
 # At least one storage class exists in K8s
 if having_category kube ; then
-    test "$(kubectl get storageclasses 2>&1 | wc -l)" -gt 1
+    test ! "$(kubectl get storageclasses | grep "No resources found.")"
```

Comment:
The error is written to STDERR, so I don't think your `grep` will ever see it. I think the `2>&1` redirection has to stay.

Reply:
Sure, I changed it. Thanks.